src/create-cache.js

```diff
@@ -4,6 +4,130 @@ import log4js from 'log4js';
 
 const logger = log4js.getLogger('dreamworld.redis-cache.cache-manager');
 
+/**
+ * It adds given keys into `__keys__` Set.
+ * @param {*} cache 
+ * @param {Array} keys 
+ */
+const _setKeys = (cache, keys) => {
```

Comment:
Rename this to `_addKeys`.

Whether this logic works or not depends on whether the given prefix is auto-applied to the `sadd` command or not. So, it's trial and error scenario. You might need to change this if it doesn't work. e.g. `__keys__` ( a root-level set is created in redis). Don't forget to verify the redis data through redis-cli.